metricbeat/module/windows/perfmon/_meta/fields.yml

```diff
@@ -5,22 +5,14 @@
       type: group
       description: >
         Grouping of different counters
-      fields: 
-        - name: group
+      fields:
```

Comment:
The fields for the `perfmon` metricset are all dynamic so I think none of these fields (counters, alias, query) should be present in fields.yml.

@ruflin Is there anything special that should put into fields.yml for this case? I think we just need empty `perfmon` group?